.circleci/config.yml

```diff
@@ -0,0 +1,22 @@
+version: 2
+jobs:
+ build:
+   machine: true
+   steps:
+     - checkout
+     - run: |
+         cd stand
+         (docker-compose up)&
```

Comment:
it is probably better to run this in executor, maybe ignoring `docker-compose` - just manually call docker